pom.xml

```diff
@@ -27,6 +27,11 @@
 				<configuration>
 					<source>${java.version}</source>
 					<target>${java.version}</target>
+					<!-- needed to compile with JDK 16 and lombok -->
+					<fork>true</fork>
```

Comment:
This needs to either be removed or set to false. As we are not compiling an executable. [More Info](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#fork)